_plugins/auto_logs_markup.rb

```diff
@@ -52,7 +63,7 @@
         "<td class='log-lineno'><a href='#l-#{index}'>#{lineno}</a></td>" \
         "<td class='log-time'>#{time}</td>" \
         "<td>" \
-          "<span class='log-nick'>&lt;#{nick}&gt;</span>" \
+          "<span class='log-nick' color='#{color}'>&lt;#{nick}&gt;</span>" \
```

Comment:
```suggestion
          "<span class='log-nick' style=\'color:#{color}'>&lt;#{nick}&gt;</span>" \
```

I think this needs to be a style tag. Using an html color attribute doesn't seem to work for me.

Reply:
Thanks -- done.